apps/hire/src/app/settings/_components/edit-board-sheet.tsx

```
@@ -0,0 +1,136 @@
+        </SheetHeader>
+        <form
+          onSubmit={(e) => {
+            e.preventDefault();
+            e.stopPropagation();
+            void form.handleSubmit();
+          }}
+          className="space-y-6 py-6"
+        >
+          <form.AppField
+            name="name"
+            children={(field) => <field.FieldInput label="Board Name" />}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)